jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/RecommendedServerIssue.kt

```diff
@@ -0,0 +1,36 @@
+package org.jellyfin.sdk.discovery
+
+import org.jellyfin.sdk.model.ServerVersion
+
+public sealed interface RecommendedServerIssue {
+	/**
+	 * No system information found. This could happen due too networking issues or an invalid server address.
+	 */
+	public data class MissingSystemInfo(public val throwable: Throwable?) : RecommendedServerIssue
+
+	/**
+	 * The product name in the system information is incorrect.
+	 */
+	public data class InvalidProductName(public val productName: String?) : RecommendedServerIssue
+
+	/**
+	 * No version found in system information.
+	 */
+	public object MissingVersion : RecommendedServerIssue
```

Comment:
I'd suggest adding a `toString()` method to this object.

Reply:
why? it's an internal thing, you're not supposed to println it

Reply:
Then why does the cli sample print it? :eyes:
I'd also recommend it for consistency - the other data classes define a toString too.

Reply:
cli is not a production tool :p

Reply:
You're right, but as I said, consistency ^^